util/new_keymap.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+"""
+Script to make a new keymap for a keyboard of your choosing.
+This script automates the copying of the default keymap into your own keymap.
+"""
+import argparse
+import logging
+import os
+import sys
+
+from distutils.dir_util import copy_tree
+
+
+usage_message = "./new_keymap.py <keyboard_path> <username>\n" + \
```

Comment:
This can be removed since the script no longer takes these arguments.

Reply:
oh right, good catch. Thank you